provision/acc_provision/acc_provision.py

```diff
@@ -540,11 +540,15 @@ def Raise(exception):
              else Raise(Exception("Invalid value: %s; "
                                   "Expected one of: {%s}" %
                                   (x, ','.join(y)))))))
+    isname = lambda x: (1 < len(x) < 32) & \
+        x[0].isalpha() & x.replace('_', '').isalnum() \
+        if x else Raise(Exception("Missing option"))
     get = lambda t: functools.reduce(lambda x, y: x and x.get(y), t, config)
 
     checks = {
         # ACI config
         "aci_config/system_id": (get(("aci_config", "system_id")), required),
```

Comment:
I think you could just change this check instead of adding another one that doesn't correspond to any field in the config.

"aci_config/system_id": (get(("aci_config", "system_id")), lambda x: required(x) and isname(x)),

Reply:
Got it. I mis-understood the last feedback. Updated now.